src/renderer/components/ft-list-channel/ft-list-channel.js

```diff
@@ -40,7 +43,11 @@ export default Vue.extend({
       this.thumbnail = this.data.avatar
       this.channelName = this.data.name
       this.id = this.data.channel_id
-      this.subscriberCount = this.data.followers.toString().replace(/\B(?=(\d{3})+(?!\d))/g, ',')
+      if (this.hideChannelSubscriptions) {
+        this.subscriberCount = null
+      } else {
+        this.subscriberCount = this.data.followers.toString().replace(/\B(?=(\d{3})+(?!\d))/g, ',')
```

Comment:
this line is the cause of my listed issue number 5. followers doesn't appear to be an accessible field in this scope